modules/apps/adaptive-media/adaptive-media-web/src/main/resources/META-INF/resources/adaptive_media/js/utils/entryIcons.es.js

```diff
@@ -15,19 +15,15 @@
 export function disableEntryIcon(element) {
 	element.parentElement.classList.add('disabled');
 
-	element.setAttribute('data-href', element.getAttribute('href'));
 	element.setAttribute('data-onclick', element.getAttribute('onclick'));
 
-	element.removeAttribute('href');
 	element.removeAttribute('onclick');
 }
 
 export function enableEntryIcon(element) {
 	element.parentElement.classList.remove('disabled');
 
-	element.setAttribute('href', element.getAttribute('data-href'));
 	element.setAttribute('onclick', element.getAttribute('data-onclick'));
 
-	element.removeAttribute('data-href');
```

Comment:
Unsure of these changes, but since we don't intend to use `data-href` for icons, it seemed appropriate to remove this.

Reply:
Makes sense to me 👍